README.md

```diff
@@ -73,3 +52,4 @@ Give a ⭐️ if you like this project!
 This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Kindly update the MIT link to the correct link in your readme, this link currently redirects to page not found, and please delete the `NOTE` comment this will make your project more professional   👍🏿 

### OPTIONAL

- [ ] Kindly add the `Contributing` section in your readme, this will give a link to your issues page 👍🏿